modules/api_reference/src/Plugin/Reference/OpenApi3.php

```diff
@@ -25,7 +25,7 @@ protected function getSchema(): string {
    * {@inheritdoc}
    */
   protected function isValid(\stdClass $data): bool {
-    return ($data->openapi ?? NULL) === '3.0.0';
+    return preg_match('/^3\.0\.[\d]+$/', ($data->openapi ?? '')) !== FALSE;
```

Comment:
Maybe preg_match() should not be run as fallback, what about 
```php
<?php

if ($data->openapi === NULL) {
  return FALSE;
}

return preg_match('/^3\.0\.[\d]+$/', $data->openapi)) !== FALSE
```

More code, but maybe more straightforward what this function do.